src/mgmt/cli_mgmt.c

```diff
@@ -549,6 +549,58 @@ int dc_mgmt_net_cfg(const char *name)
 	return rc;
 }
 
+/*
+ * Send an upcall to the agent to notify it of a clean process shutdown.
+ */
+int
+dc_mgmt_disconnect(void)
+{
+	struct drpc_alloc	 alloc = PROTO_ALLOCATOR_INIT(alloc);
+	struct drpc		*ctx;
+	Drpc__Call		*dreq;
+	Drpc__Response		*dresp;
+	int			rc;
```

Comment:
tiny nit - alignment is off between rc and alloc

Reply:
Fixed the alignment